### ISSUE TITLE
PUL Logo should show focus indicator on tab

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/lux-design-system#444]
### Steps to recreate
1. Go to the [LuxLibraryHeader component in the Lux Styleguide](https://pulibrary.github.io/lux-design-system/#!/Components/LuxLibraryHeader)
2. Tab until the "Leave and Travel Requests" shows a focus indicator
3. Shift-tab and notice that although your focus is on the library logo, no focus indicator appears

### WCAG Success Criterion
[2.4.7 - focus visible](https://www.w3.org/TR/WCAG22/#focus-visible)

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Disorienting

### Recommended fix
Ensure focus indicator is visible for keyboard navigation, both on its own and within the context of the LuxLibraryHeader

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
While the change probably needs to be made in the [LuxLibraryLogo](https://pulibrary.github.io/lux-design-system/#!/Components/LuxLibraryLogo), it's harder to see the behavior there.
